### ISSUE TITLE
Exception on resume from suspend

### ISSUE THREAD

**Issue by dkogias** [pithos/pithos#392]
```
pithos.py:879:song_text:AttributeError: 'Song' object has no attribute 'bitrate'

Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/pithos/pithos.py", line 801, in on_gst_error
    self.next_song()
  File "/usr/lib/python3.5/site-packages/pithos/pithos.py", line 572, in next_song
    self.start_song(self.current_song_index + 1)
  File "/usr/lib/python3.5/site-packages/pithos/pithos.py", line 548, in start_song
    self.update_song_row()
  File "/usr/lib/python3.5/site-packages/pithos/pithos.py", line 915, in update_song_row
    self.songs_model[song.index][1] = self.song_text(song)
  File "/usr/lib/python3.5/site-packages/pithos/pithos.py", line 879, in song_text
    if not song.bitrate is None:
AttributeError: 'Song' object has no attribute 'bitrate'

Local variables in innermost frame:
artist: 'Cantoma'
album: 'Cantoma (Deluxe)'
self: <pithos.PithosWindow object at 0x7f9945baaea0 (PithosWindow at 0x55652d1ae2b0)>
title: 'Marisi'
song: <pithos.pandora.pandora.Song POg1ybpxbMAFTK14y1zsBe2e-e5ElRto2XG6gy18lI4dhUDGPWNlIHwc7Rsr6f0MQmzxfgyizTrJK2VkL-oHVZg "Marisi" by "Cantoma" from "Cantoma (Deluxe)">
msg: []
```

**Comment by JasonLG1979** [comment]
That doesn't have anything to do with Wayland, and the bitrate error is a bit of a red herring. The bitrate bug has been fixed. The problem is that playlists are only good for an hour(Pandora's doing) if you suspend or otherwise pause Pithos for longer any pending songs will no longer be valid. That has also been fixed in the sense that Pithos won't crash it will just skip the invalid songs and get a new playlist.

**Comment by JasonLG1979** [comment]
Try the latest source and see if it fixes your problem.

**Comment by TingPing** [comment]
I'd suggest the Flatpak as an easy way to test the latest: https://github.com/pithos/pithos/wiki/Flatpak

**Comment by TingPing** [comment]
Anyway was fixed by 5bc9671b0056ed7f49a2ed29a1b8e39e90c00369

**Comment by hackel** [comment]
Just built the latest master and can confirm this issue is resolved, thanks!
